tests/conftest.py

```diff
@@ -758,3 +758,17 @@ def android_scan(clean_db: None) -> None:
         session.add(vulnerability4)
         session.commit()
     yield scan
+
+
+@pytest.fixture
+def agent_group() -> models.AgentGroup:
+    """Create dummy agent group."""
+    with models.Database() as session:
+        agent_group = models.AgentGroup(
+            name="Agent Group 1",
+            description="Agent Group 1 description",
+            created_time=datetime.datetime.now(),
```

Comment:
agents

Reply:
why ?, this is to test the deletion of agent_group, the mapping between agents and agent group is tested on the graphql api tests and the models tests